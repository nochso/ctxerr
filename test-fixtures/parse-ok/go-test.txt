	parser_test.go:1: shrug