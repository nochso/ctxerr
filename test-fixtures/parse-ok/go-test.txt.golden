shrug
parser_test.go:1:
1 | package ctxerr
  | ^^^^^^^^^^^^^^
